### PR TITLE
fix network bandwidth limitation description

### PR DESCRIPTION
**Pull request by YangKeao**
Signed-off-by: YangKeao <yangkeao@chunibyo.icu>

increase the number of limit in the example, and add description about the suggested value for `limit` in document.

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/version-2.0.6/simulate-network-chaos-on-kubernetes.md [hunk 1]
```diff
@@ -226,4 +226,4 @@ NetworkChaos 用于模拟集群中网络故障的场景，目前支持以下几
 | peakrate | uint64 | `bucket` 的最大消耗率    |        | 否       | 1     |
 | minburst | uint32 | `peakrate bucket` 的大小 |        | 否       | 1     |
 
-其中 `peakrate` 和 `minburst` 通常情况下不需要设置。如果需要进一步了解这些字段的含义，可以参考 [tc-tbf 文档](https://man7.org/linux/man-pages/man8/tc-tbf.8.html)
+其中 `peakrate` 和 `minburst` 通常情况下不需要设置。如果需要进一步了解这些字段的含义，可以参考 [tc-tbf 文档](https://man7.org/linux/man-pages/man8/tc-tbf.8.html).`limit` 建议至少设置为 `2 * rate * latency`，其中 `latency` 为发送者到目标的延迟。过小的 limit 会造成高丢包率，从而影响 TCP 连接的吞吐。
```
Comment (WangXiangUSTC) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.0.6/simulate-network-chaos-on-kubernetes.md:229]: 是否需要说明一下 latency 的值如何获取？

Comment (YangKeao) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.0.6/simulate-network-chaos-on-kubernetes.md]: 好，简单提一下可以使用 `ping` 命令进行估算。

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**